backend/usb-libusb.c

```diff
@@ -692,7 +695,10 @@ close_device(usb_printer_t *printer)	/* I - Printer */
       * If we have changed the configuration from one valid configuration
       * to another, restore the old one
       */
-      if (printer->origconf > 0 && printer->origconf != number2)
+      if (printer->quirks & USB_QUIRK_NO_ALT_SET)
```

Comment:
Let's not log this every time since we already log when  we restore the configuration.